prime-router/metadata/fhir_mapping/hl7/message/ORU_R01.yml

```diff
@@ -27,6 +27,13 @@ resources:
       - .NTE
       - .NK1
 
+  - resourceName: RelatedPerson
```

Comment:
Question for reviewers here: both `segments/NK1/RelatedPerson` and `common/patient-contact` can and will populate NK1, and the HL7->FHIR mapping currently populates both RelatedPerson and patient.contact. Should we keep it this way, or should we remove the FHIR->Hl7 patient.contact->NK1 mapping? It might make a difference for FHIR senders if they only send us the NK1 info in one FHIR location or the other and not both as we will do. For max flexibility it does seem like our HL7->FHIR mapping should continue to populate both.

Reply:
I was thinking we would remove the patient.contact->NK1 mapping in favor of the RelatedPerson->NK1 and only support that option for our FHIR senders. The NK1 -> Patient mapping has some inconsistencies won't really work.